atomicapp/nulecule/base.py

```diff
@@ -548,4 +550,9 @@ def _get_artifact_paths_for_path(self, path):
                 if dir_child.startswith('.') or os.path.isdir(dir_child_path):
                     continue
                 artifact_paths.append(dir_child_path)
+        elif os.listdir(path) == []:
+            raise NuleculeException("Artifact directory %s is empty" % path)
```

Comment:
this is dead code. If it is a directory the it will pass the 2nd if and won't ever get to the 3rd one right?